react/src/components/BookFilters/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import search_image from '../../assets/search.svg';
+
+class BookFilters extends Component {
+  constructor(props) {
+    super(props);
+    this.handleFilterTextChange = this.handleFilterTextChange.bind(this);
+    this.handleFilterFieldChange = this.handleFilterFieldChange.bind(this);
+  }
+  handleFilterTextChange(e) {
+    this.props.onFilterTextChange(e.target.value);
+  }
+  handleFilterFieldChange(e) {
+    this.props.onFilterFieldChange(e.target.value);
+  }
+
+  render() {
+    return (
+      <form className="bookFilters">
+        <select
+        defaultValue=""
+        value={this.props.filterField}
+        onChange={this.handleFilterFieldChange}
+        >
+          <option value="" disabled hidden>Seleccionar filtro</option>
```

Comment:
It would be great to extract this into a `strings.js` file. What do you think ?

Reply:
What would you extract to strings.js? What would strings.js contain?